package/health_daemon/src/baseline_monitor.c

```diff
@@ -86,29 +90,33 @@ static int sbp_msg_baseline_ecef_callback(health_monitor_t* monitor, u16 sender_
   return result;
 }
 
-static int baseline_threshold_rate_limiting_timer_callback(health_monitor_t* monitor, void *context)
+static int baseline_threshold_rate_limiting_timer_callback(health_monitor_t *monitor,
+                                                           void *context)
 {
   (void)context;
   log_fn_t log_fn = health_monitor_get_log(monitor);
   if (baseline_monitor_ctx.past_threshold) {
-    log_fn(LOG_WARNING, "Baseline Distance Over Threshold: %.4fm", baseline_monitor_ctx.distance_over_threshold);
+    log_fn(LOG_WARNING,
+           "Baseline Distance Over Threshold: %.4fm",
+           baseline_monitor_ctx.distance_over_threshold);
     baseline_monitor_ctx.past_threshold = false;
   }
 
   return 0;
 }
 
-int baseline_threshold_health_monitor_init(health_ctx_t* health_ctx)
+int baseline_threshold_health_monitor_init(health_ctx_t *health_ctx)
 {
   baseline_monitor = health_monitor_create();
   if (baseline_monitor == NULL) {
     return -1;
   }
 
-  return health_monitor_init(baseline_monitor, health_ctx,
-                            SBP_MSG_BASELINE_ECEF, sbp_msg_baseline_ecef_callback,
-                            BASELINE_ALERT_RATE_LIMIT, baseline_threshold_rate_limiting_timer_callback,
-                            NULL);
+  return health_monitor_init(
```

Comment:
I think we typically column align arguments for code style.  I think we could stand to use clang-format on the piksi_buildroot code base (same as other repos).

Reply:
Ok you caught me on that one. Was trying to avoid 80 char length overage.